src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/GroupBurdenEstimatesController.kt

```diff
@@ -63,6 +63,13 @@ open class GroupBurdenEstimatesController(
                 data
         )
 
+        // Then, maybe close the burden estimate set
+        val keepOpen = context.queryParams("keepOpen")?.toBoolean() ?: true
```

Comment:
so this is defaulting to true instead of false, sorry for not spotting it

Reply:
oh wait, that was by design? there's a test: if keepOpen is not provided, burden estimate set is left open